node/tests/TestUtilities.ts

```diff
@@ -475,9 +477,30 @@ export async function transactionTest(
     baseTransaction.lset(key5, 0, field + "3");
     responseData.push(['lset(key5, 0, field + "3")', "OK"]);
     baseTransaction.lrange(key5, 0, -1);
+    args.push([field + "3", field + "2"]);
```

Comment:
There are no `args` anymore - delete all references

Reply:
yup it is fixed 👍